_includes/components/footer.html

```diff
@@ -38,13 +38,13 @@
             {% if logo.url %}
         <a href="{{ logo.url }}">
             {% endif %}
-          <img class="usa-footer-logo-img" src="{{ logo.src | relative_url }}" alt="{{ logo.alt }}" {% if logo.width %}width="{{ logo.width }}"{% endif %} {% if logo.height %}height="{{ logo.height }}"{% endif %}>
+          <<img class="usa-footer-logo-img" src="{% if logo.external == true %}{{ logo.src }}{% else %}{{ logo.src | relative_url }}{% endif %}" alt="{{ logo.alt }}"{% if logo.width %}width="{{ logo.width }}"{% endif %}{% if logo.height %}height="{{ logo.height }}"{% endif %}>
```

Comment:
Looks like there's an extra `<` at the beginning of the `img` tag

Reply:
I think since this is boolean `{% if logo.external == true %}` it could probably just be `{% if logo.external %}`

Reply:
Ah - thanks (I had the change local but need to push it up) 👍